client/components/webcamModule.js

```diff
@@ -0,0 +1,114 @@
+angular.module('in-your-face')
+.controller('webcamModuleCtrl', function($scope) {
+  console.log("123");
+  var _video = null;
+  var patData = null;
+  var self = this;
+
+  this.patOpts = {x: 0, y: 0, w: 25, h: 25};
+
+  // Setup a channel to receive a video property
+  // with a reference to the video element
+  // See the HTML binding in main.html
+  this.channel = {};
+  //
+  // this.webcamError = false;
+  // this.onError = function (err) {
+  //     $scope.$apply(
+  //         function() {
+  //             self.webcamError = err;
+  //         }
+  //     );
+  // };
+  //
+  this.onSuccess = function () {
+          // The video element contains the captured camera data
+    _video = this.channel.video;
+    $scope.$apply(function() {
+      _video.width = 540;
```

Comment:
The dynamic size assignment will be implement after modifying the surrounding divs, and i am planing to do that after the technical assessment.